downstream/modules/eda/con-decision-frameworks.adoc

```diff
@@ -0,0 +1,15 @@
+[id="decision-frameworks"]
+
+= Decision Frameworks
+
+
+{EDAName} includes, by default, an ansible.eda collection, which contains some sample sources, event filters and rulebooks. All the collections, ansible rulebooks and their dependencies use a Decision Environment, which is an image that can be run on either Podman or Kubernetes.
+
+In Decision Environments, sources, which are typically Python code, are distributed through ansible-collections. They inject external events into a rulebook for processing. The rulebook consists of the following:
+
+* The python interpreter
+* Java Runtime Environment
```

Comment:
We should say why we require Java here:   "Java Runtime Environment for Drools rule engine"